docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/linux-packages.rst

```diff
@@ -84,6 +84,17 @@ Install PostgreSQL using ``apt`` or ``yum``
       sudo -u postgres psql postgres
       postgres=# \password postgres
 
+#. The authorization methods enabled vary depending upon the provider of your PostgreSQL
+   distribution. Ensure that an appropriate authorization method is configured in ``pg_hba.conf`` to
```

Comment:
Minor nit, but this is more authentication (the "hba" in hba.conf is "host-based authentication").  Authorization is done internally through grants.
```suggestion
#. The authentication methods enabled vary depending upon the provider of your PostgreSQL
   distribution. Ensure that an appropriate authentication method is configured in ``pg_hba.conf`` to
```

Reply:
Yup.  Good catch.